packages/components-angular/package.json

```diff
@@ -27,12 +29,14 @@
     "zone.js": "0.15.0"
   },
   "devDependencies": {
+    "@swisspost/design-system-components-angular": "workspace:9.0.0-next.33",
```

Comment:
This is only used for the consumer-app and does not end up in the lib package.json.